app/src/pages/inside/projectSettingsPageContainer/content/elements/fieldElement/fieldElement.jsx

```diff
@@ -23,14 +23,14 @@ import styles from './fieldElement.scss';
 const cx = classNames.bind(styles);
 
 export const FieldElement = (props) => {
-  const { label, description, children, className, ...rest } = props;
+  const { label, description, children, className, childrenClassName, ...rest } = props;
   return (
     <div className={cx('wrapper', className)}>
       {label ? (
         <>
           <span className={cx('label')}>{label}</span>
-          <span className={cx('description')}>{description}</span>
-          <div className={cx('wrapped-children')}>
+          {description && <span className={cx('description')}>{description}</span>}
+          <div className={cx('wrapped-children', childrenClassName)}>
```

Comment:
Maybe we don't needed `wrapped-children` at all it looks like a mistake I suggest remove class `wrapped-children` and leave only childrenClassName